app/src/utils/Utils.ts

```diff
@@ -252,3 +251,30 @@ export const getFormattedIdentitySource = (identitySource: SYSTEM_IDENTITY_SOURC
       return null;
   }
 };
+declare global {
+  interface Array<T> {
+    alphabetizeObjects(property: keyof T): T[];
+  }
+}
+
+/**
+ * For a given property, alphabetize an array of objects
+ *
+ * @param {string} property This function will alphabetize the array of objects on the given property
+ * @returns {any[]} Returns an alphabetized array of objects
+ */
+// eslint-disable-next-line no-extend-native
+Array.prototype.alphabetizeObjects = function <T extends { [key: string]: any }>(property: keyof T): T[] {
+  return this.sort((a: T, b: T) => {
+    const nameA = a[property].toUpperCase();
+    const nameB = b[property].toUpperCase();
+
+    if (nameA < nameB) {
```

Comment:
Could simply return `nameB - nameA`? That's how I've seen alphabetical sort implemented, at least

Reply:
good call, simplified that